subprojects/android-test/test-annotations/src/main/kotlin/com/avito/android/test/annotations/TestType.kt

```diff
@@ -1,40 +1,46 @@
 package com.avito.android.test.annotations
 
+@Deprecated("2020.3.2; use @Kind")
```

Comment:
Maybe we need to use `replaceWith` argument?

Reply:
i decided to revert `@Kind` annotation for now, and use old ones, so can't determine replaceWith value